jekyll/_cci2/testing-orbs.md

```diff
@@ -128,7 +128,14 @@ circleci local execute --job shellcheck/check
 ### Review
 {: #review }
 
-The orb-tools orb includes a job `orb-tools/review` which will run a suite of tests against your orb designed to find opportunities to implement best practices and improve the quality of the orb. The "review" job was modeled closely after _ShellCheck_, and operates based on a list of rules called "RC" Review Checks. Each "RC" code corresponds to a specific rule, which can optionally be ignored.
+The orb-tools orb includes a job `orb-tools/review` which will run a suite of tests against your orb designed to find opportunities to implement best practices and improve the quality of the orb. The "review" job was modeled closely after _ShellCheck_, and operates based on a list of rules called "RC" Review Checks. Each "RC" code corresponds to a specific rule, which can optionally be ignored using the `exclude` parameter in your config.yaml file.
+
+Example:
+```yaml
+      - orb-tools/review:
+          filters: *filters
+          exclude: RC006,RC007
+```
```

Comment:
```suggestion
```yaml
version: 2.1

orbs:
  orb-tools: circleci/orb-tools@11.1

workflows:
  my-workflow:
    jobs:
      - orb-tools/review:
          exclude: RC006,RC007
```
```

Reply:
@KyleTryon could you check the syntax here is correct?